apps/backend/src/parse/parse.controller.ts

```diff
@@ -111,135 +110,14 @@ export class ParseController {
-    if (!daily) {
-      daily = await this.parseService.createNewDaily(rules, new Date());
-    }
-
-    try {
-      // FileType is based on the filename (from Parser) or from the endpoint body
-      if (fileType === FileTypes.SBC_SALES) {
-        this.appLogger.log('Parse and store SBC Sales in DB...', fileName);
-        const garmsSales: TransactionEntity[] =
-          await this.parseService.parseGarmsFile(contents, fileName); // validating step
-        const fileToSave = await this.parseService.saveFileUploaded({
-          sourceFileType: fileType,
-          sourceFileName: fileName,
-          sourceFileLength: garmsSales.length,
-          dailyUpload: daily,
-        });
-        this.appLogger.log(`Transaction count: ${garmsSales.length}`);
-        await this.transactionService.saveTransactions(
-          garmsSales.map((sale) => ({
-            ...sale,
-            fileUploadedEntity: fileToSave,
-          }))
-        );
-      }
-
-      if (fileType === FileTypes.TDI17) {
-        this.appLogger.log('Parse and store TDI17 in DB...', fileName);
-        const cashDeposits = await this.parseService.parseTDICashFile(
-          fileName,
-          program,
-          file.buffer
-        ); // validating step
-        const fileToSave = await this.parseService.saveFileUploaded({
-          sourceFileType: fileType,
-          sourceFileName: fileName,
-          sourceFileLength: cashDeposits.length,
-          dailyUpload: daily,
-        });
-        this.appLogger.log(`Cash Deposits count: ${cashDeposits.length}`);
-        await this.cashDepositService.saveCashDepositEntities(
-          cashDeposits.map((deposit) => ({
-            ...deposit,
-            fileUploadedEntity: fileToSave,
-          }))
-        );
-      }
-
-      if (fileType === FileTypes.TDI34) {
-        this.appLogger.log('Parse and store TDI34 in DB...', fileName);
-        const posEntities = await this.parseService.parseTDICardsFile(
-          fileName,
-          program,
-          file.buffer
-        );
-        const fileToSave = await this.parseService.saveFileUploaded({
-          sourceFileType: fileType,
-          sourceFileName: fileName,
-          sourceFileLength: posEntities.length,
-          dailyUpload: daily,
-        });
-        this.appLogger.log(`POS Deposits count: ${posEntities.length}`);
-        await this.posDepositService.savePOSDepositEntities(
-          posEntities.map((deposit) => ({
-            ...deposit,
-            fileUploadedEntity: fileToSave,
-            timestamp: deposit.timestamp,
-          }))
-        );
-      }
-    } catch (err: unknown) {
-      const message =
-        err instanceof Error
-          ? err.message
-          : `Error with processing ${fileName}`;
-      this.appLogger.log(message);
-      throw new BadRequestException({ message });
-    }
+    //TODO call parse service to process file event
+    return {
+      fileName,
```

Comment:
This may be more appropriate to return?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501